monai/handlers/stats_handler.py

```diff
@@ -174,7 +174,7 @@ def _default_iteration_print(self, engine: Engine) -> None:
         """
         Execute iteration log operation based on Ignite engine.state data.
         Print the values from Ignite state.logs dict.
-        Default behavior is to print loss from output[1], skip if output[1] is not loss.
+        Default behavior is to print loss from output[0] as output is decollated list.
```

Comment:
I think we need some additional documentation to explain this...
```suggestion
        The default behavior is to print loss from output[0] as output is a decollated list.
```

Reply:
enhanced the doc-strings in the latest commit.

Thanks.